server/persistence/memory/memory.go

```diff
@@ -11,3 +11,9 @@ func New() persistence.DB {
 		getInteractionService(),
 	)
 }
+
+func Clear() {
+	clearGameService()
+	clearPlayerService()
+	clearInteractionService()
+}
```

Comment:
💡 instead of adding new clear funcs, I think it would be sufficient (and more accurate/threadsafe) to just nil out the pointers for each service. That is:

```go
func Clear() {
	gservice = nil
	iservice = nil
	pservice = nil
}
```

as written, we're just clearing out the maps within each, but that _could_ be race-y.